hyperwallet/api.py

```diff
@@ -2791,4 +2792,41 @@ def createTransferSpendBackRefund(self,
             params
         )
 
-        return TransferRefunds(response)
\ No newline at end of file
+        return TransferRefunds(response)
+
+    '''
+
+    Create User Transition
+
+    '''
+
+    def createUserStatusTransition(self,
+                                   userToken=None,
+                             data=None):
+        '''
+        Create User Transition.
```

Comment:
Correct the parameter details and put it above the method, not inside the method.

Reply:
as per your comments has been resolved